setup.php

```diff
@@ -20,67 +20,114 @@
+								<p>The recommended way of doing this is to edit the file first,
+								then use this page, and finally use the software in a real
+								environment.</p>
+							</div>
 						</div>
 					</div>
-				</div>
-				<div class="row">
-					<div class="panel panel-default col-xs-offset-1 col-xs-10">
-						<div class="panel-heading">
-							<h3 class="panel-title">Administrator Credentials</h3>
-						</div>
-						<div class="panel-body">
-							<p><?php echo $ProjectName ?> currently has an Administrator
-							which can configure the parameters of the software. Please
-							enter the user credentials below.</p>
-							<p>Make sure the credentials you use are secure and cannot
-							be guessed by employees and/or customers who may obtain
-							access to the service.</p>
-							<div class="row">
-								<div class="form-group col-xs-offset-1 col-xs-10">
-									<label for="username">Username:</label>
-									<input type="text" class="form-control" id="username" name="username"></input>
-								</div>
-								<div class="form-group col-xs-offset-1 col-xs-10">
-									<label for="password">Password:</label>
-									<input type="password" class="form-control" id="password" name="password"></input>
-								</div>
-								<div class="form-group col-xs-offset-1 col-xs-10">
-									<label for="password2">Repeat Password:</label>
-									<input type="password2" class="form-control" id="password2" name="password2"></input>
+					<div class="row">
+						<div class="panel panel-default col-xs-offset-1 col-xs-10">
+							<div class="panel-heading">
+								<h3 class="panel-title">Administrator Credentials</h3>
+							</div>
+							<div class="panel-body">
+								<p><?php echo $ProjectName ?> currently has an Administrator
+								which can configure the parameters of the software. Please
+								enter the user credentials below.</p>
+								<p>Make sure the credentials you use are secure and cannot
+								be guessed by employees and/or customers who may obtain
+								access to the service.</p>
+								<div class="row">
+									<div class="form-group col-xs-offset-1 col-xs-10">
+										<label for="username">Username:</label>
+										<input type="text" class="form-control" id="username" name="username"></input>
+									</div>
+									<div class="form-group col-xs-offset-1 col-xs-10">
+										<label for="password">Password:</label>
+										<input type="password" class="form-control" id="password" name="password"></input>
+									</div>
+									<div class="form-group col-xs-offset-1 col-xs-10">
+										<label for="password2">Repeat Password:</label>
+										<input type="password" class="form-control" id="password2" name="password2"></input>
+									</div>
 								</div>
-
+									<button type="submit" class="btn btn-info col-xs-offset-1 center-block">Submit</button>
 							</div>
 						</div>
 					</div>
+				</form>
+			<?php }else{
+				class MyDB extends SQLite3{
```

Comment:
This part of code looks like something that will be repeated throughout the codebase and only used once. If that's the case, can you move it to a file that's included where needed such as `database.php`, which will be `require_once()`'d every time a database transaction is needed? 

That way we can then support multiple database backends such as SQLite3, MySQL, PostgreSQL, etc.